src/olympia/git/management/commands/git_extraction.py

```diff
@@ -1,145 +0,0 @@
-from django.conf import settings
-from django.core.management.base import BaseCommand
-
-import waffle
-from celery import chain
-
-import olympia.core.logger
-from olympia import amo
-from olympia.amo.decorators import use_primary_db
-from olympia.files.utils import lock
-from olympia.git.models import GitExtractionEntry
-from olympia.git.tasks import (
-    continue_git_extraction,
-    extract_versions_to_git,
-    on_extraction_error,
-    remove_git_extraction_entry,
-)
-
-
-log = olympia.core.logger.getLogger('z.git.git_extraction')
-
-# Number of versions to extract in a single task. If you change this value,
-# please adjust the soft time limit of the `extract_versions_to_git` task.
-# See: https://github.com/mozilla/addons-server/issues/14104
-BATCH_SIZE = 10
-# Name of the lock() used.
-LOCK_NAME = 'git-extraction'
-# Name of the waffle switch.
-SWITCH_NAME = 'enable-git-extraction-cron'
-# The number of entries to process when the command is invoked.
-LIMIT = 20
-
-
-class Command(BaseCommand):
```

Comment:
Yes, good point. I should also file a ticket for SRE because `scripts/crontab/crontab.tpl` is actually obsolete these days, SRE are managing our crons in a separate repos

Reply:
https://mozilla-hub.atlassian.net/browse/SVCSE-2340

Reply:
https://github.com/mozilla/addons-server/pull/22873/commits/586239ab3aa3a4c84816e621f98049a472a87785